Xamarin.PropertyEditing.Mac/Controls/Custom/DrawingExtensions.cs

```diff
@@ -0,0 +1,130 @@
+using System;
+using AppKit;
+using CoreGraphics;
+using CoreImage;
+using Xamarin.PropertyEditing.Drawing;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	public static class DrawingExtensions
```

Comment:
If it's helpers for a platform implementation like this, let's keep it `internal`.